.circleci/config.yml

```
@@ -687,7 +716,7 @@ jobs:
           path: src/test/regress/tmp_check/master/log
       - store_artifacts:
           name: 'Save worker1 log'
-          path: src/test/regress/tmp_check/worker.57637/log
+          path: src/test/regress/tmp_cßheck/worker.57637/log
```

Seems like a mistake

```suggestion
          path: src/test/regress/tmp_check/worker.57637/log
```